devices.json

```diff
@@ -2,5 +2,6 @@
   "nexus5x": "01ca4c64db5d1014",
   "nexus6p": "KYVKK16119600056",
   "xcompact": "BH904E9V5P",
```

Comment:
In my pull request I have implemented a feature that allows users to specify a path for this file outside of this repo using a config option. I think this is a nice feature, as it allows users to use Android Runner without modifying any files inside of the repo itself. 

This is also a plus if Android Runner is gonna be packaged and distributed through a package manager for example.

Reply:
This has now been implemented